src/rows.rs

```diff
@@ -146,82 +144,6 @@ pub struct CoinRow<L2: Layer2Coin> {
     pub layer2: Vec<L2>,
 }
 
-impl<L2: Layer2Cache> WalletCache<L2> {
```

Comment:
Why do we delete these whole block? I use it in my projects, and there may be other users